stable/xray/README.md

```diff
@@ -121,8 +121,13 @@ Removing a **helm** release is done with
 
 ```bash
 # Remove the Xray services and data tools
+
+#On helm v2:
 helm delete --purge xray
 
+#On helm v3:
+helm delete xray -n xray
```

Comment:
@shahiinn  Pls use —namespace instead of -n, so the same command works for both v2 and v3 helm